lime/graphics/format/JPEG.hx

```diff
@@ -28,8 +29,9 @@ class JPEG {
 		var bufferData:Dynamic = lime_jpeg_decode_bytes (bytes, decodeData);
 		
 		if (bufferData != null) {
-			
-			var buffer = new ImageBuffer (bufferData.data, bufferData.width, bufferData.height, bufferData.bpp, bufferData.format);
+
+			var u8a = new UInt8Array (@:privateAccess new Bytes (bufferData.data.length, bufferData.data.b));
+			var buffer = new ImageBuffer (u8a, bufferData.width, bufferData.height, bufferData.bpp, bufferData.format);
 			buffer.transparent = bufferData.transparent;
```

Comment:
This is required if the texture is to be used by any processing function ( copyPixels, copyChannels).
But it also breaks if decode data is false.
A fix would be 
```
var u8a:UInt8Array = null;
if(decodeData){
    u8a = new ....
}
```

This is valid for the 4 changes. Apart from that, those changes need to be merge :-)